test/validators/movie.test.js

```diff
@@ -0,0 +1,54 @@
+'use strict';
+
+const Joi = require('joi');
+
+const MovieValidator = require('../../lib/validators/movie');
+
+describe('movie validator', () => {
+
+  describe('title', () => {
+
+    it('is required', () => {
+      const payload = {};
+      const result = Joi.validate(payload, MovieValidator);
```

Comment:
nit: add line after consts

Reply:
This was maybe ambiguous -- I think jb was more saying to keep the `const` declarations together, and then separate them from the `expects` with a newline!

Reply:
oh ya that was definitely my bad! when i clicked to comment, github highlighted the 3 previous lines (instead of the line after, which i wanted to showcase). i'll be more specific next time 😅

as for newlines, we want to use them to define logical groups of code or enhance readability. This means we tend to group `const`s together, have a newline, group `let` variables together, have another newline, etc.

Reply:
ok this makes more sense. just changed this